app/src/main/res/values/dimens.xml

```diff
@@ -141,4 +141,7 @@
     <dimen name="fab_normal_margin">8dp</dimen>
     <dimen name="fab_mini_margin">16dp</dimen>
 
+    <dimen name="remote_icon_size">84dp</dimen>
```

Comment:
These are not needed as Android will pick the ones closest to the specified smallest width. I even think these will never be used as we use smallest width specific dimensions.